.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+### Summary
+[PLEASE REMOVE] Describe the goal of this PR and any add any relevant context.
+
+[PLEASE REMOVE] If this PR closes an issue, please add a `Fixes #<issue-id>` line.
+
+[PLEASE REMOVE] If this PR introduces a fix or feature that should be the upcoming release notes, please add the following "Release notes: <area>" label.
```

Comment:
"add the following label" makes it sound like a) there's only one label, and b) there should be a list of labels here. Maybe `please add the appropriate "Release notes: <area>" label.`